### DISCUSSION TITLE
Intents

### DISCUSSION THREAD

**Discussion by kukucsi** [FuseFairy/DiscordBot-EdgeGPT#116] [Q&A]
Why does the bot need all 3 intents? I'm just curious which intent does what and why does the bot need it.

**Comment by FuseFairy** [comment]
https://github.com/FuseFairy/DiscordBot-EdgeGPT/issues/77
In fact, bot only need the MESSAGE CONTENT INTENT, I think I will change in the next version.

**Comment by herowarriors0** [comment]
And approximately when will that update happen? I'm not saying it as an urge, I'm just curious.

**Comment by FuseFairy** [comment]
I've been too busy before, so I'm just starting to get some time to update today, and I plan to create a threat when a user uses /bing . This way everyone goes on their own chat channel,

**Comment by herowarriors0** [comment]
Can I disable the other two intents if I use the recently published updated bot? Or the bot still needs all three intents?

**Comment by FuseFairy** [comment]
You can disable the other two intents
